From: John Rigby <jrigby@freescale.com>
Date: Thu, 24 Aug 2006 12:50:09 -0600
Subject: [PATCH] mpc5200 nogetospace

The SNDCTL_DSP_GETOSPACE ioctl does not work
correctly with the ac97 sound driver on Media5200.
This patch turns off the usage of this ioctl
on this platform.

Signed-off-by: John Rigby <jrigby@freescale.com>

---

 qt2/src/kernel/qsoundqss_qws.cpp |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

f500b7122a83a0b84f432b40775bcebeef6c085f
diff --git a/qt2/src/kernel/qsoundqss_qws.cpp b/qt2/src/kernel/qsoundqss_qws.cpp
index fdacc3a..ab4fb1c 100644
--- a/qt2/src/kernel/qsoundqss_qws.cpp
+++ b/qt2/src/kernel/qsoundqss_qws.cpp
@@ -855,7 +855,11 @@ #endif
 	fd = -1;
 	active.setAutoDelete(TRUE);
 	unwritten = 0;
+#ifdef QT_QWS_MPC5200
+	can_GETOSPACE = FALSE;
+#else
 	can_GETOSPACE = TRUE;
+#endif
     }
 
 signals:
-- 
1.3.3

